components/application-operator/charts/application/templates/deployment.yaml

```diff
@@ -136,4 +136,8 @@ spec:
         ports:
         - containerPort: {{ .Values.eventService.deployment.args.externalAPIPort }}
           name: http-api-port
-{{- end }}
```

Comment:
https://github.com/kyma-project/kyma/pull/10030#issuecomment-742592539

Reply:
Those files are added a couple of days ago, before my PR could be approved. https://github.com/kyma-project/kyma/commit/b8d434f80405dce9855e184374e8877c30facaa7#diff-8bc71f3198f84116be60fa4a367cee495c497904a85a84d3efba46467814987c